### DISCUSSION TITLE
504 Gateway Time Out ?

### DISCUSSION THREAD

**Discussion by darth2602** [lllyasviel/Fooocus#2533] [Q&A] [answered]
Hello, on Google Colab Version , since this morning when I click on the link that launches the Fooocus interface, nothing happens and finally I get a message 504 Gateway Time Out .... What's going on? Does it work for me? (I have a very fast fiber connection, everything works fine to access the web)

**Comment by joostvanderree** [comment]
I'm running against the same issue right now. Are you running it locally or through Google Colab like me?

**Reply by xgwxx** [reply]
I have the same problem I get 504 Gateway Time Out messages

**Reply by joostvanderree** [reply, marked as answer]
It's a Gradio live issue, we just need to wait until it's operational again https://status.gradio.app

**Reply by darth2602** [reply]
Yes Google Colab (I've updated my post)

**Comment by joostvanderree** [comment]
I'm not very technical, but perhaps it has to do that Gradio link seems to be down?
<img width="1050" alt="Screenshot 2024-03-15 at 11 04 04" src="https://github.com/lllyasviel/Fooocus/assets/7425877/169a3a08-e4f1-4ee6-a589-bd7e2eec62da">
